e2e/verifier-configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: verifier
+data:
+  COORDINATOR_API_URL: 'http://coordinator-service:8080'
+  COORDINATOR_AUTH_TYPE: celo
+  COORDINATOR_SEED_FILE: '/mnt/secrets/seed'
```

Comment:
Verifier doesn't need a seed.

Reply:
Oh right.  That's a pre-existing oops with the coordinator-client.  I'll spin up another PR to address that and then circle back around to this one.

Reply:
FYI, https://github.com/celo-org/snark-setup-coordinator/pull/6